src/applications/personalization/profile/tests/e2e/nametag.cypress.spec.js

```diff
@@ -6,7 +6,7 @@ import disabilityRating from '@@profile/tests/fixtures/disability-rating-success
 import error401 from '@@profile/tests/fixtures/401.json';
 import error500 from '@@profile/tests/fixtures/500.json';
 
-import mockUser from '../fixtures/users/user';
+import { mockUser } from '../fixtures/users/user';
```

Comment:
suggestion: clearly since this test doesn't even need this mock user data, just delete this import entirely. And change line 21 to `cy.login()`

Reply:
and thanks for fixing this goof 🤦